ofl/pangmenzhengheading/DESCRIPTION.en_us.html

```diff
@@ -0,0 +1,8 @@
+<p>庞门正道标题体，由设计公众号"<a href="https://mp.weixin.qq.com/mp/profile_ext?action=home&__biz=MzA5NjIzNjgxNw==&scene=124#wechat_redirect">《庞门正道》</a>"以及13位字体设计师联合研发。</p>
```

Comment:
We also need an English description

Reply:
OK, I will resubmit as soon as possible.